imagetest/test_suites/disk/disk_resize_test.go

```diff
@@ -52,7 +53,8 @@ func TestDiskResize(t *testing.T) {
 }
 
 func getDiskSize() (int64, error) {
-	fstatOut, err := exec.Command("df", "-B1", "--output=size", "/").CombinedOutput()
+	// NOTE(sejalsharma): / to /var.
+	fstatOut, err := exec.Command("df", "-B1", "--output=size", "/var").CombinedOutput()
```

Comment:
I think this one should still be `/`, since we're measuring disk size instead of writing anything here

Reply:
The reason the disk size is measuring from "/var" is because we changed the boot marker file installation: "markerFile = "/var/boot-marker". This is due to the read-only root file system of COS, requiring file installations to be in /var. So the test would only pass if we measured in /var too.

Reply:
The boot marker file doesn't affect reading the root disk size, it should still read root. This (`/var`) would still work but it would be needlessly confusing as `df` will look up the filesystem `/var` is on (`/` unless something weird happens) and just read the file size of `/` anyway.

Reply:
When I switch the df command back to "/", the test fails. I tried running the df command on a COS VM and it is measuring /var as having a different size than /, I've attached a snippet below. I believe this might be due to [the way COS manages partitions](https://cloud.google.com/container-optimized-os/docs/concepts/disks-and-filesystem).
<img width="914" alt="Screenshot 2024-02-06 at 10 22 22 AM" src="https://github.com/GoogleCloudPlatform/guest-test-infra/assets/145373269/22b0b7ae-bcaa-4861-a551-5fd86e7a2476">

Reply:
Oh, I understand now, thanks. Measuring `/var` only for COS and root for everyone else would be ideal then.

Reply:
Thanks! I can work on adding this as COS specific only (strings.HasPrefix(image, "cos-")) and update the code.

Reply:
Updated the code to be COS specific